pos_pricelist/views/pos_config_view.xml

```diff
@@ -25,7 +25,7 @@
                             widget="many2many_tags"/>
                         <field
                             name="pricelist_id"
-                            domain="[('id', 'in', available_pricelist_ids)]"
+                            domain="[('id', 'in', available_pricelist_ids and available_pricelist_ids[0] and available_pricelist_ids[0][2] or False)]"
```

Comment:
This domain will return only one id ? Or did  I miss something ? I think it should return all the ids available in the field available_pricelist_ids.

Reply:
No, it returns all the existing ones, as it's stored as [(6, 0, [<ids>])], so [0][2] accesses [<ids>], but I didn't know this works on the JS part. @chienandalu is it working?

Reply:
Yes, it is :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/5040182/42449598-916c4370-8381-11e8-913f-176f93e8af3d.png)

Reply:
In v11 the right operator is more transparent, anyway (this bug was due to the literal copy of the view in this backport)

Reply:
Thanks for the explanation